src/features/content-answers/ContentAnswersModalChat.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import throttle from 'lodash/throttle';
+
+import Answer from './Answer';
+import Question from './Question';
+import WelcomeMessage from './WelcomeMessage';
+import { QuestionType } from './ContentAnswersModal';
+// @ts-ignore: no ts definition
+// eslint-disable-next-line import/named
+import { User } from '../../../common/types/core';
+
+import './ContentAnswersModalChat.scss';
+
+type Props = {
+    currentUser: User;
+    fileName: string;
+    handleScrollToBottom: (behavior: string) => void;
+    isLoading: boolean;
+    questions: QuestionType[];
+};
+
+const ContentAnswersModalChat = ({ currentUser, fileName, handleScrollToBottom, isLoading, questions }: Props) => {
+    const throttledScrollToBottom = throttle(handleScrollToBottom, 200);
+
+    return (
+        <div className="bdl-ContentAnswersModalChat">
+            <WelcomeMessage fileName={fileName} />
+            <ul>
+                {questions &&
```

Comment:
questions.length?

Reply:
I simplified the condition check to `questions?.map`. But I think we need to leave the <ul> tag outside since we don't want to map on that and instead we want to map <li> tag for each pair of question and answer inside of the <ul> tag.